README.md

```diff
@@ -47,8 +47,8 @@ __Note__: If installation stops due to billing account errors, set up the billin
 ### Next Steps
 
 * Explore your Sandbox deployment and its [architecture](#Service-Overview)
-* Follow the [User Guide](/docs/README.md) to start using Ops Management
-* Learn more about Ops Management using [Code Labs](https://codelabs.developers.google.com/s/results?q=Monitoring)
+* Follow the User Guide to start using [Cloud Operations](https://cloud-ops-sandbox.dev/docs/user-guide/learn-cloud-learn-cloud-operations/)
```

Comment:
shouldn't we still have a link to the user guide here? (and below)

Reply:
This the address of the new user guide